og.module

```diff
@@ -303,6 +303,38 @@ function og_entity_type_alter(array &$entity_types) {
   }
 }
 
+/**
+ * Implements hook_theme().
+ */
+function og_theme($existing, $type, $theme, $path) {
+  return [
+    'og_member_count' => [
+      'variables' => [
+        'count' => 0,
+        'membership_states' => [],
+        'group' => NULL,
+      ],
+    ],
+  ];
+}
+
+/**
+ * Prepares variables for member count templates.
+ *
+ * Default template: og-member-count.html.twig.
+ *
+ * @param array $variables
+ *   An associative array containing:
```

Comment:
The array contains also the `membership_states` key but this is not detailed below.